examples/function_memory_game/src/components/game_status_board.rs

```diff
@@ -0,0 +1,39 @@
+use crate::constant::Status;
+use yew::prelude::*;
+use yew::{function_component, html, Properties};
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props {
+    pub status: Status,
+    pub sec_past: u32,
+    pub on_reset: Callback<()>,
+}
+
+#[function_component]
+pub fn GameStatusBoard(props: &Props) -> Html {
+    let get_content = {
+        let onclick = props.on_reset.reform(move |e: MouseEvent| {
+            e.stop_propagation();
+            e.prevent_default();
+        });
+
+        match props.status {
+            Status::Ready => html! {
+                <span >{"Ready"}</span>
```

Comment:
```suggestion
                <span>{"Ready"}</span>
```

Reply:
@hamza1311  Would you please share how you got such error? I ran with `cargo make lint`, there is no any warning. Thanks

Reply:
This is not a warning, just a formatting issue. The warnings are in other comment where @voidpumpkin explained how to run the lints